frontend/src/libs/components/header/libs/helpers/get-burger-menu-items.helper.ts

```diff
@@ -0,0 +1,20 @@
+import { type BurgerMenuItem } from '~/libs/types/types.js';
+import { UserGroupKey } from '~/packages/users/libs/enums/enums.js';
+
+import { BURGER_MENU_ITEMS } from '../../constants/constants.js';
+
+const getBurgerMenuItems = (group?: string): BurgerMenuItem[] => {
```

Comment:
```suggestion
const getBurgerMenuItems = (group: string | null): BurgerMenuItem[] => {
```
Defined arguments are preferred one.
`getBurgerMenuItems(user?.group.key ?? null)`